sematic/api_client.py

```diff
@@ -314,9 +327,12 @@ def _notify_event(namespace: str, event: str, payload: Any = None):
     backoff=2,
     jitter=0.1,
 )
-def _get(endpoint) -> Any:
+def _get(endpoint: str, content: bool = False) -> Any:
```

Comment:
```suggestion
def _get(endpoint: str, decode_json: bool = True) -> Any:
```

Looking at the var name `content` I have no idea what it does, but `decode_json` seems a lot more explanatory.

Reply:
Done.